lib/Locale/locale_service.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:language_code/language_code.dart';
+
+abstract class LocaleService {
+  String getLocale();
+  void setLocale(String? locale);
+}
+
+class LocaleServiceImpl implements LocaleService {
+  static String? locale;
+
+  static String getLocaleName() {
+    String deviceLocale = LanguageCode.rawLocale.toString();
```

Comment:
shouldn't we set `locale` value here?
otherwise we're activating the language code again with no reason

Reply:
The formating that is recieved from LanguageCode is different from the one we have been using. I.E he_IL instead of he.